### PR TITLE
update docs text and replace most screenshots with GUI updates

### PR DESCRIPTION
**Pull request by KZzizzle**
## What do these changes do?

Updates all documentation text. Replace screenshots with new ones for the updated GUI.

## Related issue number

closes https://github.com/ITISFoundation/osparc-issues/issues/574

## Checklist

- [x] I think the documentation is well written and up to date
- [ ] Do NOTE have very large media files (e.g. photos, videos, etc)
- [x] My spell-checker passes
- [ ] If you write a new section, i take ownership adding my user in .github/CODEOWNERS

### CODE REVIEW

File: docs/platform_introduction/appmode.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# App Mode
+
+*o²S²PARC* provides another viewing option for a guided experience in exploring a **Study**. The idea behind the "App Mode" is to provide a way to step through the Services in a **Study** in a linear fashion, much like passing back and forth between slides of a slide presentation. This mode shows only the settings of Computational Services or the *Interactive View* of Interactive Services. To set up you Study for App Mode, you will first need to select which Services to display and in which order. Not all Services need to be displayed in App Mode. However, if certain Services must be run in order to access others (e.g. A Computational Service must run in order to create the data for a 2D Plot Service to plot), it will be run automatically during App Mode execution.2D
+
+## App Mode Setup
+1. Click on the empty space in your **Pipline Workspace** to see the **Study** information in the **Workbench** Secondary Column. Near the bottom, you will see "App Mode" - under this section, click the ```Edit``` button.
```
Comment (odeimaiz) [inline docs/platform_introduction/appmode.md:6]:
```suggestion
1. Click on the empty space in your **Pipeline Workspace** to see the **Study** information in the **Workbench** Secondary Column. Near the bottom, you will see "App Mode" - under this section, click the ```Edit``` button. 
```

File: docs/platform_introduction/studies.md [hunk 2]
```diff
@@ -1,43 +1,55 @@
...
-
-5. **Service:** each Service is visible in the pipeline workspace as a box with a name, input port, and/or an output port.
-6. **Run Button:** for pipelines containing computational Services (such as *isolve*), this button is used to run execute the pipeline.
-
+1. **Primary Column:** 
+    * In Nodes List tab, you can see the list of Services that are included in the Study and rename or delete them. To delete a Service, select the Service and then click the Delete button on the menu bar. You can also rename a Service by selecting a Service and clicking the rename button on the menu bar. 
+    * In the Storage tab, you can browse all the data from any study you have created.
+2. **Secondary Column:** 
+    * When no Service is selected, this tab will show the Study metadata
+    * When a Service is selected as in the case of the image above, the inputs and output parameters of that service will be show in separate tabs.
```
Comment (odeimaiz) [inline docs/platform_introduction/studies.md:39]:
```suggestion
    * When a Service is selected as in the case of the image above, the inputs and output parameters of that service will be shown in separate tabs.
```

File: docs/platform_introduction/studies.md [hunk 3]
```diff
@@ -1,43 +1,55 @@
...
+3. **Workbench Views:** 
+    * **Pipeline**: this is the default view of the pipeline, showing services and their connections
+    * **Interactive**: when an Interactive Service is selected in the pipeline view or in the primary column, this tab will show the GUI for the Interactive Service
+    * **Logger**: this tab will display all messages pertinent to the running of your Study. These messages can be filtered by keywords/keyphrases entered into the “Filter” field.
+4. **Run/Stop** for pipelines containing Computational Services, the **Run** button is used to run execute the pipeline while the **Stop** button can abort a running pipeline.
```
Comment (odeimaiz) [inline docs/platform_introduction/studies.md:44]:
```suggestion
4. **Run/Stop** for pipelines containing Computational Services, the **Run** button is used to execute the pipeline while the **Stop** button can abort a running pipeline.
```

File: docs/tutorials/opencor.md [hunk 4]
```diff
@@ -7,27 +7,31 @@ OpenCOR is a software projects aimed at providing a modelling environment for Ce
...
 
-    ![opencor_flow](https://user-images.githubusercontent.com/32800795/61585149-741ae400-ab55-11e9-8a40-605192454acf.JPG ':size=600%')
+2. After creating your own instance of the OpenCOR Template or access it through a MAP-CORE map, you will see a simple flowchart consisting of two Services. The o²S²PARC-opencor Service will do the full simulation with your own input parameters.
 
-2. The input parameters you can choose will be the type of stimulation and normalized amplitude. You can access these by double clicking on the o²S²PARC-opencor Service.
+    ![opencor_flow](https://user-images.githubusercontent.com/28002886/153856064-5b871a9f-cc76-48d4-bff7-de8933578c29.png ':size=600%')
 
-3. You can get a little more information about the inputs by hovering over the value adjustment arrows on the right of each field.
+3. The input parameters you can choose will be the type of stimulation and normalized amplitude. You can access these by double clicking on the o²S²PARC-opencor Service.
```
Comment (odeimaiz) [inline docs/tutorials/opencor.md:16]:
```suggestion
3. The input parameters you can choose will be the type of stimulation and normalized amplitude. You can access these by clicking on the o²S²PARC-opencor Service.
```

**Review by odeimaiz — APPROVED** [review]
THANKS 🤗

File: docs/platform_introduction/appmode.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
+# App Mode
+
+*o²S²PARC* provides another viewing option for a guided experience in exploring a **Study**. The idea behind the "App Mode" is to provide a way to step through the Services in a **Study** in a linear fashion, much like passing back and forth between slides of a slide presentation. This mode shows only the settings of Computational Services or the *Interactive View* of Interactive Services. To set up you Study for App Mode, you will first need to select which Services to display and in which order. Not all Services need to be displayed in App Mode. However, if certain Services must be run in order to access others (e.g. A Computational Service must run in order to create the data for a 2D Plot Service to plot), it will be run automatically during App Mode execution.2D
+
+## App Mode Setup
+1. Click on the empty space in your **Pipeline Workspace** to see the **Study** information in the **Workbench** Secondary Column. Near the bottom, you will see "App Mode" - under this section, click the ```Edit``` button.
```
Comment (pcrespov) [inline docs/platform_introduction/appmode.md:6]: I get a bit confused with the terms "pipeline workspace" and "workbench". What is the difference between them?

File: docs/platform_introduction/services.md [hunk 6]
```diff
@@ -11,4 +11,20 @@ The pipeline can be built up as a directed acyclic graph (DAG) where the lines b
...
+
+In terms of viewing, Interactive services will have a special view in the Workbench through the *Interactive* View tab ![interactive](https://user-images.githubusercontent.com/28002886/153265444-6638c003-d90c-4a34-b751-cb0193879993.png ':size=25'). You may occasionally want to maximize this view in order to have a larger working area. To do this, you can click on the ```maximize``` button on the upper right hand corner of the *Interactive View*.
+
+![minimize](https://user-images.githubusercontent.com/28002886/153834101-ed179672-6c23-440f-ac0b-971426f16e39.png ':size=300') 
+
+To regain access the other **Workbench** options such as inspecting Service inputs and outputs or changing the Pipeline, you can minimize the *Interactive View* by clicking on the minimize button on the upper right hand corner.
```
Comment (pcrespov) [inline docs/platform_introduction/services.md:24]: MINOR: why you use sometimes capitalize words in the middle of a sentence  e.g. ``changing the *P*ipeline``  and moreover sometimes is bold and sometimes not.
Look in this sentence the words Study/ies, tab and service/s: `` Study in your **Studies** tab with the single Service inside the Study. ... o go through the **Services** Tab.``

File: docs/platform_introduction/studies.md [hunk 7]
```diff
@@ -1,43 +1,55 @@
...
-6. **Services:** allows you to view the Services and their versions that are a part of this Study. You will also be able to see if there are newer versions of these Services available in the Service Catalog.
-7. **Duplicate** allows you to create a copy of that Study which will be renamed with a timestamp.
-8. **Export** allows you to download the Study as a *.osparc* file which can then be imported back onto the platform by yourself or anyone who has that file
-9. **Publish as Template** if selected, you will be allowed to create a copy of the Study in its current state which will appear as a Template Study in the [Discover](docs/platform_introduction/Discover.md) tab for users you share the Template with.* 
-10. **Delete** as you may guess, this will delete the particular Study, with a confirmation popup. 
+1. **Select:** allows you to select this Study and/or others, primarily useful for deletion.
+2. **Rename:** allows you to rename the study
+<!-- 3. **Duplicate** allows you to create a copy of that Study which will be renamed with a timestamp. -->
```
Comment (pcrespov) [inline docs/platform_introduction/studies.md:18]: We will activate those soon.

TIP: in markdown, if you leave all the enumerations as  ``1. `` you do not need to adjust them anymore when you add/remove items

File: docs/platform_introduction/appmode.md (hunk 5 above)
Comment (KZzizzle) [reply docs/platform_introduction/appmode.md:6]: I named the area that the pipeline lives in as the "pipeline workspace" (excludes the primary/secondary columns) - if you think it's too much, I can replace all of them by "workbench".

File: docs/platform_introduction/studies.md [hunk 8]
```diff
@@ -1,43 +1,55 @@
...
+<!-- 4. **Export** allows you to download the Study as a *.osparc* file which can then be imported back onto the platform by yourself or anyone who has that file -->
+3. **More Options:** brings up a modal window where you are able to view additional options for a Study including its description, sharing settings, quality metrics, Services contained, etc. - editing may also be possible if you have appropriate permissions.
+
+    ![moreoptions](https://user-images.githubusercontent.com/28002886/153412343-7a7cd0f1-3790-4bc6-afba-46f061efb8bf.png ':size=400')
```
[the remark this answers is not in the document]
Comment (odeimaiz) [reply docs/platform_introduction/studies.md:22]: Not that I know of

[the remark this answers is not in the document]
Comment (KZzizzle) [reply docs/platform_introduction/studies.md:22]: so far it has not? I have them as comments so I doubt they disappear https://github.com/KZzizzle/osparc_images/issues/2

File: docs/platform_introduction/services.md (hunk 6 above)
Comment (KZzizzle) [reply docs/platform_introduction/services.md:24]: usually I do this to indicated "protected" osparc words. It's true though that I'm not always consistent... will double check later. Do you prefer non-bold and non capitalized always?

File: docs/platform_introduction/appmode.md (hunk 5 above)
Comment (pcrespov) [reply docs/platform_introduction/appmode.md:6]: no, no ... i was not sure. all good

Comment (pcrespov) [reply docs/platform_introduction/appmode.md:6]: no, no ... i was not sure. all good

File: docs/platform_introduction/services.md (hunk 6 above)
Comment (pcrespov) [reply docs/platform_introduction/services.md:24]: no preference :-) thx

**Review by pcrespov — APPROVED** [review]
:+1:

**Pull request merged**